packages/expo-router/plugin/options.json

```diff
@@ -136,6 +136,23 @@
             }
           }
         },
+        "headers": {
+          "description": "A list of header values that are sent with every response from the server",
+          "type": "object",
+          "additionalProperties": {
```

Comment:
future: We could validate `propertyNames` here in the future, I guess, but I'm not sure if the error would be as useful. Let's leave that for a future PR maybe